packages/backend/src/server/api/stream/channels/main.ts

```diff
@@ -25,6 +31,8 @@ export default class extends Channel {
 					break;
 				}
 				case 'mention': {
+					if (isInstanceMuted(body, new Set<string>(this.userProfile?.mutedInstances ?? []))) return;
+
```

Comment:
```suggestion
					if (isInstanceMuted(data.body, new Set<string>(this.userProfile?.mutedInstances ?? []))) return;

```

TypeScriptの仕様上、Destructuring assignmentするとswitch-case内でtypeがおかしくなってしまうので出来れば避けてください。:pray: